dwave/cloud/hybrid.py

```diff
@@ -0,0 +1,71 @@
+# Copyright 2020 D-Wave Systems Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Interface to hybrid :term:`sampler`s available through the D-Wave API.
+"""
+
+from __future__ import absolute_import
+
+from dwave.cloud.client import Client as BaseClient
+from dwave.cloud.solver import UnstructuredSolver as Solver
```

Comment:
Where is this import being used?

Reply:
In user's code, potentially. This is to bring in `Solver` in this namespace.

Reply:
The reason it caught my eye was the loss of the unstructured characteristic, but I guess you're not worried about confusion in applications that compare structured and unstructured solvers?

Reply:
That's a valid concern, I guess I was going for symmetry here, because `StructuredSolver` is already aliased as `Solver` in qpu/sw ns.